src/app/_user-connection/PageHeaderLeftSlotStrategy.tsx

```diff
@@ -0,0 +1,25 @@
+'use client'
+import { usePathname } from 'next/navigation'
+
+/**
+ * Uses Strategy Algorithm to know when to render the left components which vary from page to page
+ * @constructor
+ */
+export function PageHeaderLeftSlotStrategy() {
+  const pathname = usePathname()
+  const LeftComponent = getLeftComponentForRoute(pathname)
+
+  return LeftComponent ? LeftComponent : null
```

Comment:
Why not simply `return LeftComponent`?

Reply:
Looks cleaner to me @rodrigoncalves even tho I can change it, not a problem

Reply:
@TravellerOnTheRun I am not, please re-check